lib/plugins/inventory.js

```diff
@@ -434,6 +434,13 @@ function inject (bot, { version, hideErrors }) {
       }
       return
     }
+    if (windowClickQueue.length > 0 && Math.max(...windowClickQueue.map(x => x.id)) < actionId) {
+      if (!hideErrors) {
+        console.log('WARNING : ignoring transaction where actionId > max action id in windowClickQueue and dropping all of windowClickQueue')
+        windowClickQueue.pop()
```

Comment:
move `windowClickQueue.pop()` out of `if (!hideErrors)`

Reply:
done :)